docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -733,3 +697,88 @@ Optionally update your App with the new prop usage:
 #### CDN CSP Policy
 
 The 2.x version of the `FaceLivenessDetector` has updated the default CDN for TensorFlow and Blazeface to a new URL. If your application has an existing CSP policy, update your policy to allow https://cdn.liveness.rekognition.amazonaws.com. Please look over the [Liveness Config](../connected-components/liveness/customization#facelivenessdetectorconfig) section for more information.
+
+
+## `@aws-amplify/ui-react-notifications`
```

Comment:
Concerned abt the discoverability of the migration changes with moving `@aws-amplify/ui-react-notifications` and `@aws-amplify/ui-react-storage` so far down in the guide?

Reply:
I could add a note in the react section that points you to them? I just went alphabetically

Reply:
Adding package links at the top of the page would make sense imo for at least the short term. @dbanksdesign You have any thoughts?

Reply:
Yeah, I like the idea of package links, though I suppose they are in the TOC?

Reply:
Yup h2s and h3s are in the TOC, package names being h2s they are in there: https://ui.docs.amplify.aws/react/getting-started/migration

<img width="1507" alt="CleanShot 2023-11-15 at 20 51 10@2x" src="https://github.com/aws-amplify/amplify-ui/assets/321279/20dd322e-188f-47ff-a532-927e482b97dd">

If we wanted we could add a list of links with package names in the body?

Reply:
> If we wanted we could add a list of links with package names in the body?

This is what i was thinking, imo would improve discoveravbility